### PR TITLE
[receiver/windowsperfcounters] Clarify current treatment of the `_Total` instance

### LINKED ISSUE #29054: [receiver/windowsperfcounters] Instance wildcard doesn't capture _Total instance
### Component(s)

receiver/windowsperfcounters

## Description

When the instance is set to "*" the user wants to capture all instances of a counter like `\Processor(*)\%Process Time`. However, the wildcard doesn't capture the `_Total` instance.

## Steps to Reproduce

Run the collector using a config like this one:
```yaml
receivers:
  windowsperfcounters:
    metrics:
      processor.time:
        description: CPU active and idle time
        unit: "%"
        gauge:
    collection_interval: 30s
    perfcounters:
      - object: "Processor"
        instances:
           - "*"
        counters:
          - name: "% Processor Time"
            metric: processor.time
exporters:
  debug:
    verbosity: detailed
service:
  pipelines:
    metrics:
      receivers: [windowsperfcounters]
      exporters: [debug]
```

## Expected Result

That the `_Total` instance is also captured when the wildcard is used.

## Actual Result

The `_Total` instance is not captured.

If the user tries to workaround by forcing the capture by explicitly adding `_Total` to the instances, i.e.:
```yaml
    perfcounters:
      - object: "Processor"
        instances: ["_Total", "*"]
        counters:
          - name: "% Processor Time"
            metric: processor.time
```
It still doesn't work because internally if the instances contain "*" they are treated as if only "*" was specified.

The way to workaround the issue is to create two receivers: one to capture `_Total` and one to capture the other instances, the later one using the wildcard.

### Collector version

22d86de838

## Environment
OS: Windows 10, Windows Server 2022
Compiler(if manually compiled): go1.21.1 windows/amd64

### OpenTelemetry Collector configuration

```yaml
receivers:
  windowsperfcounters:
    metrics:
      processor.time:
        description: CPU active and idle time
        unit: "%"
        gauge:
    collection_interval: 30s
    perfcounters:
      - object: "Processor"
        instances:
           - "*"
        counters:
          - name: "% Processor Time"
            metric: processor.time
exporters:
  debug:
    verbosity: detailed
service:
  pipelines:
    metrics:
      receivers: [windowsperfcounters]
      exporters: [debug]
```

### Log output

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by pjanotti**
**Description:**
Clarifies the current treatment of the `_Total` instances. 

**Link to tracking Issue:**
Fix #29054

**Testing:**
Manually validated the configuration added to the README.md

**Documentation:**
Described the behavior of the `_Total` instance as currently implemented.

### CODE REVIEW

File: receiver/windowsperfcountersreceiver/README.md [hunk 1]
```diff
@@ -58,17 +55,45 @@ windowsperfcounters:
...
+  perfcounters:
+    - object: "Processor"
+      instances:
+          - "_Total"
+      counters:
+        - name: "% Processor Time"
+          metric: processor.time.total
+```
+
+> [!WARNING]
```
Comment (pjanotti) [inline receiver/windowsperfcountersreceiver/README.md:92]: This actually gives an idea to how better handle counters that don't use `_Total` for their aggregation. We can have a configuration called something like `aggregation_name`, that by default is `_Total`, if someone is using a counter that uses a different name for the aggregation instance we let them specify so the problem of double counting can be easily solved.

Comment (alxbl) [reply receiver/windowsperfcountersreceiver/README.md:92]: The aggregation_name would be used to decide which instance to drop from a wildcard collection? That makes sense to me.

**Review by alxbl — APPROVED** [review]

File: receiver/windowsperfcountersreceiver/README.md (hunk 1 above)
Comment (pjanotti) [reply receiver/windowsperfcountersreceiver/README.md:92]: Yes, so it keeps the current default behavior, but, if some user wants to drop the `_Global_` they add that to the configuration.

**Review by crobert-1 — COMMENTED** [review]
Thanks for adding more documentation here @pjanotti! Just had some formatting comments/questions.

File: receiver/windowsperfcountersreceiver/README.md [hunk 2]
```diff
@@ -58,17 +55,45 @@ windowsperfcounters:
...
 Not specified | This is the only valid value if the counter has no instances
 `"*"` | All instances
-`"_Total"` | The "total" instance
+`"_Total"` | The "total" instance, that aggregates the values of all other instances. See below for its special treatment.
 `"instance1"` | A single instance
 `["instance1", "instance2", ...]` | A set of instances
-`["_Total", "instance1", "instance2", ...]` | A set of instances including the "total" instance
+
+> [!NOTE]
+> Aggregation counter and the behavior of the `_Total` instance
```
[the remark this answers is not in the document]
Comment (pjanotti) [reply receiver/windowsperfcountersreceiver/README.md:70]: You are right a title would be better.

File: receiver/windowsperfcountersreceiver/README.md [hunk 3]
```diff
@@ -58,17 +55,45 @@ windowsperfcounters:
           recreate_query: <true or false>
 ```
 
-*Note `instances` can have several special values depending on the type of
-counter:
+> [!NOTE]
```
[the remark this answers is not in the document]
Comment (pjanotti) [reply receiver/windowsperfcountersreceiver/README.md]: Opted make it a header.

**Review by crobert-1 — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]

**Pull request merged**